ambuda/static/css/prosemirror-example-setup.css

```diff
@@ -0,0 +1,15 @@
+/* Originally from:
+https://github.com/ProseMirror/prosemirror-example-setup/blob/master/style/style.css
+https://github.com/ProseMirror/prosemirror-gapcursor/blob/master/style/gapcursor.css
+*/
+
+#editor,
+.editor {
```

Comment:
rather than these, let's just use tailwind classes on the editor:

```
<div id="editor" class="bg-white text-black">
```

Likewise for `.Prosemirror`

Reply:
Just removed this (background white and text black: they're the default anyway), but unfortunately the `.Prosemirror` is harder to remove. The reason is the `.Prosemirror` element is dynamically inserted as a *child* of this `#editor` element, so if we want to address it with Tailwind, it's… actually possible (`https://github.com/tailwindlabs/tailwindcss/pull/8299`) but ugly.